content/TrueCommand/TCGettingStarted/InitialConfig.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Creating the Initial Configuration"
+description: "This article describes how to initially configure TrueCommand."
+aliases:
+weight: 20
+tags:
+- tcinstall
+- tcadmin
+- tcconfig
+- tcresetpwd
+- scaletoptoolbar
+- coretoptoolbar
+---
+{{< toc >}}
+TrueCommand allows you to manage all your TrueNAS systems from one location. This multi-system management increases efficiency and simplifes operations. To get started with the initial configuration of TrueCommand, ensure that the following steps have been completed:
+
+* You have created the administrator account. See [Creating the Admin Account]({{< relref "/TrueCommand/TCGettingStarted/AdminAccounts.md" >}}) for more information.
+* You have created the necessary user accounts. See [Creating User Accounts]({{< relref "/TrueCommand/TCGettingStarted/UserAccounts.md" >}}).
+* You familiar with the interface toolbars and icons. Review them here [Interface Overview]({{< relref "/TrueCommand/TCGettingStarted/InterfaceOverview.md" >}}).
+* You have connected your first TrueNAS system. See [Connecting Your First TrueNAS System]({{< relref "/TrueCommand/TCGettingStarted/ConnectingTrueNAS.md" >}}) for more information.
+
+The **Administration** page, available to users with administrator permissions, displays additional system details and offers a variety of TrueCommand configuration options.
+Click the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> icon and select **Administration** to access the **Administration** page. It is organized into **About**, **Certificates**, and **Configuration** panes.
```

Comment:
We are now moving away from using panes in favor of using widgets to describe these information cards.
```suggestion
Click the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> icon and select **Administration** to access the **Administration** screen. It is organized into **About**, **Certificates**, and **Configuration** widgets (functional and information cards).
```

Reply:
Change made.